### PR TITLE
OOP LIBRARY PAIR PROGRAMMING

### PR DESCRIPTION
**Pull request by Josphat205**
## Procedure

- Extracted some pieces of logic into separate class
- Ensured that main file start the code and ask the user for the initial choice of options

### CODE REVIEW

File: lister.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  end
+
+  def self.get_people(_people)
+    puts '-' * 50
+    if @people.empty?
+      puts 'The list is empty'
+    else
+      puts 'People list:'
+      App.new.people_list
+    end
```
Comment (DammyShittu) [inline lister.rb]: - [x] Whether there is a list of people or not the app crashes when I select the option to list people. Kindly correct this.

  listern

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @Josphat205 @abdulhamiid,

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: lister.rb (hunk 1 above)
Comment (Josphat205) [reply lister.rb]: sorted out, thank you

**Review by gemmen29 — APPROVED** [review]
Hi @Josphat205, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**